src/barril/basic/fraction/_tests/test_fraction_value.py

```diff
@@ -127,3 +127,19 @@ def AssertCreateFromString(text, whole, fraction=None):
     AssertCreateFromString("33/4", 0, (33, 4))
     AssertCreateFromString("  35/4", 0, (35, 4))
     AssertCreateFromString("3 36/4", 3, (36, 4))
+
+
+def testCreateFromFloat():
+    """
+    Allow the user enter only the fraction value
+    """
+
+    def AssertCreateFromFloat(value, whole, fraction=None):
```

Comment:
`fraction` has a default of `None` but is never used? I suggest to drop the default and the condition, which simplifies this test to:

```python
def testCreateFromFloat():
    """
    Allow the user enter only the fraction value
    """
    assert FractionValue.CreateFromFloat(1.375) == FractionValue(1, (3, 8))
    assert FractionValue.CreateFromFloat(2.5) == FractionValue(2, (1, 2))
```

Reply:
Nice!